library/src/main/java/com/leinardi/android/speeddial/UiUtils.java

```diff
@@ -280,4 +281,20 @@ public static Drawable getDrawableFromBitmap(@Nullable Bitmap bitmap) {
             return new BitmapDrawable(bitmap);
```

Comment:
**Reporter**: Error Prone
**Rule**: deprecation
**Severity**: WARN
**File**: library/src/main/java/com/leinardi/android/speeddial/UiUtils.java L281

BitmapDrawable(Bitmap) in BitmapDrawable has been deprecated

return new BitmapDrawable(bitmap);^

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<1340797268>*